featuretools/tests/computational_backend/test_calculate_feature_matrix.py

```diff
@@ -1138,12 +1137,13 @@ def test_cfm_returns_original_time_indexes(es):
     if isinstance(fm, pd.DataFrame):
         instance_level_vals = fm.index.get_level_values(0).values
         time_level_vals = fm.index.get_level_values(1).values
-    else:
-        fm = to_pandas(fm, index='id')
-        instance_level_vals = fm.index
-        # Dask doesn't return time (doesn't support multi-index)
-        # Koalas does support multiindex, look into possibly supporting
-        time_level_vals = []
+    # else:
+    #     # Code needed if removing dask/koalas xfail
+    #     fm = to_pandas(fm, index='id')
+    #     instance_level_vals = fm.index
+    #     # Dask doesn't return time (doesn't support multi-index)
+    #     # Koalas does support multiindex, look into possibly supporting
+    #     time_level_vals = []
```

Comment:
Maybe we should just remove this for now as it isn't needed?

Reply:
I think it makes sense to get rid of the extra code/comments for x-failed tests, it's probably better to keep the tests clean and uncluttered with unused code.

Reply:
That's my thought as well. I'll remove this and look for any other unused blocks that can be cleaned up.